PublicSDKAPITester/RCPurchasesAPI.m

```diff
@@ -64,7 +64,7 @@ + (void)checkAPI {
     appUserID = [p appUserID];
     isAnonymous = [p isAnonymous];
     
-    RCPurchaserInfo *pi = [[RCPurchaserInfo alloc] init];
+    RCPurchaserInfo *pi = [[RCPurchaserInfo alloc] initWithData: [[NSDictionary alloc] init]];
```

Comment:
extra space after `:`

You can also do this a little more simply by using a dict literal, i.e.: 
```objective-c
RCPurchaserInfo *pi = [[RCPurchaserInfo alloc] initWithData:@{}];
```